week2/homework/src/index.js

```diff
@@ -1,3 +1,98 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+let fs = require('fs');
+let command = process.argv[2];
+let index = process.argv[3];
+let item = process.argv[4];
+
+switch (command) {
+    case 'list':
+        listValue()
+        break;
+
+    case 'add':
+        add(index);
+        break;
+
+    case 'remove':
+        Mydelete(index);
+        break;
+
+    case 'update':
+        updateValue(index, item)
+        break;
+
+    case 'reset':
+        Mydelete(0, 1);
+        break;
+
+    case 'help':
+    default:
+        printHelp();
+}
+
+function listValue() {
+    fs.readFile('./to-do.txt', 'utf8', function (error, data) {
+        if (error) {
+            console.log(error);
+        } else {
+            console.log(data);
+        }
+    })
+}
+
+function add(main) {
+    fs.appendFile('./to-do.txt', main + '\n', function (error) {
+        if (error)
+            console.log(error);
+    });
+}
+
+function Mydelete(index, reset) {
```

Comment:
Don't start your function with a capital letter: this should be `myDelete()`, or actually just `delete()`.